src/test/e2e/slider.spec.ts

```diff
@@ -1,9 +1,18 @@
 import { test, Locator, expect, Page } from '@playwright/test';
 
-const FIRST_SLIDE_TITLE_WHY_US: string = 'Open source';
-const SECOND_SLIDE_TITLE_WHY_US: string = 'Ease of setup';
-const FIRST_SLIDE_TITLE_POSSIBILITIES: string = 'Public API';
-const PUBLIC_LIBRARIES: string = 'Public Libraries';
+import t from './utils/initializeLocalization';
+
+const FIRST_SLIDE_TITLE_WHY_US: string = t('why_us.headers.header_open_source');
+const SECOND_SLIDE_TITLE_WHY_US: string = t('why_us.headers.header_ease_of_setup').replace(
+  '<br /> ',
+  ''
+);
+const FIRST_SLIDE_TITLE_POSSIBILITIES: string = t(
+  'unlimited_possibilities.cards_headings.heading_public_api'
+);
```

Comment:
@Kravalg, @TuranC. Prettier automatically breaks the line because it exceeds the printWidth value. What should be done in this case? Just shorten the variable name?

Reply:
@coderabbitai prettier automatically breaks the line because it exceeds the printWidth value. What should be done in this case? Just shorten the variable name ?

what do you think ?